src/plugins/intel_gpu/src/plugin/device_config.cpp

```diff
@@ -301,7 +288,9 @@ void Config::UpdateFromMap(const std::map<std::string, std::string>& configMap)
             } else {
                 IE_THROW(NotFound) << "Unsupported KEY_CLDNN_ENABLE_FP16_FOR_QUANTIZED_MODELS flag value: " << val;
             }
-        } else if (key.compare(GPUConfigParams::KEY_GPU_MAX_NUM_THREADS) == 0) {
+        } else if (key.compare(GPUConfigParams::KEY_GPU_MAX_NUM_THREADS) == 0 ||
+                   // Should be replaced with ov::threads_num after Properties update
+                   /* key.compare(ov::inference_num_threads.name()) == 0 */ false) {
```

Comment:
when this update should happen? What is expected here?

Reply:
This property was introduced in this https://github.com/openvinotoolkit/openvino/pull/9679 PR, but the PR has not been merged yet. So, I added this property myself to my PR in a new commit